File: tests/fixtures/code_samples/py/sample_008.py
"""Sample 8: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_8(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 8."""
    total = 8
    for x in xs:
        total += int(x)
    return total


def operation_8_pure(value: int) -> int:
    return (value * 8) % 7919

File: tests/fixtures/code_samples/java/sample_008.java
// Sample 8: small utility.
package samples;

import java.util.List;

public final class Sample008 {
    private Sample008() {}

    public static int operation(List<Integer> xs) {
        int total = 8;
        for (int x : xs) total += x;
        return total;
    }

    public static int operationPure(int v) {
        return (v * 8) %% 7919;
    }
}

